synapse/lib/cell.py

```diff
@@ -99,11 +99,11 @@ def decrfunc(func):
         def wrapped(*args, **kwargs):
 
             if args[0].user is not None and not args[0].user.isAdmin():
-                raise s_exc.AuthDeny(mesg='User is not an admin.',
-                                     user=args[0].user.name)
+                raise s_exc.AuthDeny(mesg=f'User is not an admin [{args[0].user.name}]',
+                                     user=args[0].user.iden, username=args[0].user.name)
```

Comment:
Nit: Since we're already assuming `args[0]` is `self`, you could update `wrapped` to have `self` as the first argument and change all the `args[0]` to `self`. Might be cleaner/clearer.